examples/e2e/app-router/open-next.config.ts

```diff
@@ -1,15 +1,15 @@
 import type { OpenNextConfig } from "@opennextjs/aws/types/open-next.js";
-import cache from "@opennextjs/cloudflare/kvCache";
+import incrementalCache from "@opennextjs/cloudflare/kvCache";
```

Comment:
thought: we mostly use kebab-case, maybe this should be `kv-cache`?

Reply:
agreed